src/DistortableImageOverlay.js

```diff
@@ -146,7 +153,7 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
   },
 
   isSelected() {
-    return this._selected;
+    return this._selected; // this._selected
```

Comment:
Just a comment to be removed here, thanks!

And a few other `console.log()` statements throughout as well!

Reply:
Yes, please... I will remove them all. Thanks!